### DISCUSSION TITLE
adding new features

### DISCUSSION THREAD

**Discussion by matomatusov** [slipx06/sunsynk-power-flow-card#371] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Hi,
would it be possible to add the following sensors?
1 -  add solar radiation
2 - add the status of the inverter to the card lite and compoct
3 - add cooling symbol (similar to Timer On)

![12](https://github.com/slipx06/sunsynk-power-flow-card/assets/129114957/8e355586-7e62-4b5e-9a2e-30fa9d86050f)

### Expected behaviour

adding new features

### Mode

Both

### Context / Reason

adding new features

**Comment by slipx06** [comment]
Thanks for the suggestions. 

1. Can you share some sample data (sensor and value)
2. Not planned for now. Will keep in mind.
3. Not planned for now.Will keep in mind.

**Comment by matomatusov** [comment]
Hi, I meant such a sensor.


solar radiation
![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/129114957/73f67260-7c3d-4b19-bf5d-fc906e7069cd)

**Comment by slipx06** [comment]
We need space for number in the 1000's. It might work on the lite card but not possible on the full card. Going to move this to the ideas discussion.
